src/containers/donation/index.css

```diff
@@ -0,0 +1,96 @@
+.header {
+  width: 100%;
+  padding-top: 30px;
+  text-align: center;
+  margin-top: -32px;
+}
+
+.pageMarings {
+  margin: 100px 200px;
+  font-family: "Aktiv Grotesk Corp";
+  color: #454545;
+}
+
+.address {
+  font-size: 50px;
+  font-weight: bold;
+  line-height: 60px;
+  margin-bottom: 50px;
+}
+
+.subAddress {
+  font-size: 30px;
+  font-weight: bolder;
+  line-height: 40px;
+  margin-bottom: 25px;
+  text-align: justify;
+}
+
+.textBody1 {
+  font-size: 18px;
+  font-weight: light;
+  line-height: normal;
+  color: #666666;
+  text-align: justify;
+}
+
+.textBody2 {
+  font-weight: bolder;
+  text-align: justify;
+}
+
+.quotes {
+  display: block !important;
+  margin: 0 auto;
+  padding-top: 0px;
+}
+
+.mm {
```

Comment:
I like this class name, i think you should delete one of the "m"s, one "m" is enough